### PR TITLE
feat: support search functionality in issuance dropdown

### PR DESCRIPTION
**Pull request by bhavanakarwade**
### What?
- Added search functionality in issuance using email
- Added search functionality in bulk issuance

### CODE REVIEW

**Review by pranalidhanavade — APPROVED** [review]
LG

File: src/components/Issuance/BulkIssuance.tsx [hunk 1]
```diff
@@ -133,38 +133,59 @@ const BulkIssuance = () => {
 			}
 			else if (currentSchemaType === SchemaTypes.schema_W3C && orgId && isAllSchema) {
 
-				
-				const response = await getAllSchemas(schemaListAPIParameters,currentSchemaType); 
+				let platformSchemas: ICredentials[] = [];
+				let totalItems = 0;
+				let response;
+			
+				for (let currentPage = 1; ; currentPage++) {
```
Comment (tipusinghaw) [inline src/components/Issuance/BulkIssuance.tsx:140]: Please change this logic

Comment (bhavanakarwade) [reply src/components/Issuance/BulkIssuance.tsx:140]: Resolved

File: src/components/Issuance/BulkIssuance.tsx [hunk 2]
```diff
@@ -133,38 +133,59 @@ const BulkIssuance = () => {
 			}
 			else if (currentSchemaType === SchemaTypes.schema_W3C && orgId && isAllSchema) {
 
-				
-				const response = await getAllSchemas(schemaListAPIParameters,currentSchemaType); 
+				let platformSchemas: ICredentials[] = [];
```
[the remark this answers is not in the document]
Comment (bhavanakarwade) [reply src/components/Issuance/BulkIssuance.tsx]: Resolved

**Review by pranalidhanavade — APPROVED** [review]
LG

**Review by tipusinghaw — APPROVED** [review]

**Pull request merged**